corehq/apps/hqwebapp/static/hqwebapp/font/style.scss

```diff
@@ -344,3 +344,9 @@
     content: $fcc-incomplete;
   }
 }
+
+.fcc-Manage-Case {
```

Comment:
hey @jingcheng16 it isn't favored to have caps in css classes. can you please update the font file, stylesheets, and styleguide to lowercase everything? thanks!

Reply:
nm! i see the next few commits